src/mlpack/methods/reinforcement_learning/environment/cart_pole.hpp

```diff
@@ -104,7 +104,9 @@ class CartPole
    * @param tau The time interval.
    * @param thetaThresholdRadians The maximum angle.
    * @param xThreshold The maximum position.
-   * @param doneReward Reward recieved on termination.
+   * @param doneReward Reward recieved by agent on success.
```

Comment:
picky comment!, I think we should keep the description of doneReward as "Reward recieved by agent on termination"

Reply:
I made this change since now the termination of an episode can happen because the maximum number of steps have been taken (which could be a success in cases like cart pole but a failure because in other cases) or any other termination case we have included. I changed the description to make it more clear, since it may not be evident to the user.

Reply:
Still I think termination word deals with both the cases unbiasedly. Though, no worries, you can keep it that way. :)